### PR TITLE
add an upgrade guide for v3 to changelog

### PR DESCRIPTION
**Pull request by jelhan**
Breaking changes are not that clearly structured. I think a few upgrade notes will help. Not sure if I covered everything and to be honest haven't actually done an upgrade myself yet.

### CODE REVIEW

**Comment by gilest** [comment]
Agree about structure. Will be more careful with PR names and labels now that I understand how they end up in the changelog.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,17 +1,40 @@
...
+import upload from '<app-name>/tests/helpers/upload';
+```
+
+Please change both of them to:
+
+```js
+import { selectFiles } from 'ember-file-upload/test-support';
+```
+
+The new `selectFiles` test helper expects a selector as first and one or many files as additional arguments. It does not take a file name as an argument. Please set the file name on the `File` instance:
```
Comment (gilest) [inline CHANGELOG.md]: Could possibly link to the docs here.

Comment (jelhan) [reply CHANGELOG.md]: :+1:

**Comment by jelhan** [comment]
> Will be more careful with PR names and labels now that I understand how they end up in the changelog.

It's always difficult. Especially cause it not easy to split pull requests later to have them fit better in a changelog. I think adding additional information to the changelog afterwards is totally fine.

**Pull request merged**